docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -267,7 +267,7 @@ allow_assign_grafana_admin = true
 
 ## Configure team synchronization
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise" >}}) and [Grafana Cloud](/docs/grafana-cloud/).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise" >}}) and [Grafana Cloud]({{< relref "../../../../introduction/grafana-cloud" >}}).
```

Comment:
This link is fine. When creating a link between repos, we can't use `relrefs`. We need to use absolute partial URLs.

Reply:
Currently, this URL reference is not working as expected and returns a 404.

If left unchanged, the URL resolves to
```
http://localhost:3002/docs/grafana-cloud/
```
instead of
```
http://localhost:3002/docs/grafana/latest/introduction/grafana-cloud/
```

The "dynamic" part of the URL is `latest` since this would change to a specific version that you're currently reading.

Feel free to test this in `main`! I'm more than ready to jump on a call if there are any questions.

Reply:
I wonder if it is due to local docs build only including the Grafana docs and not cloud etc docs 🤔 

I checked one of the existing references from the published docs (the link to cloud pro and advanced from [this page](https://grafana.com/docs/grafana/latest/administration/data-source-management/#data-source-permissions) is also `(/docs/grafana-cloud/)`), and it seems to resolve fine.